Holo-Host/hp-admin
# B-01231 - updated websdk enabled approach for holofuel v0.X

### Updates : 
 - adds websdk
 - updates holochainClient to call websdk when run as a holochain app in a hpos env (this is determined by global vars atm)
 - separates holofuel run and build scripts into holofuel-holochain and holofuel-holo versions
 - updates start:live script
 - adds sign-out button to holofuel header (only shows when signed in)
 - prompts for web sdk sign-in modal only if context is an anonymous user

## To Test:
 -  run `npm i` to make sure you've installed all the lasted deps (notably the new @holo-host/web-sdk and @holo-host/chaperone - for the chaperone dev server)
 - start your chaperone server by running: `npx chaperone-server --config chaperone.json`
 - start your ui by running: `npm run start:holofuel-holo-live`
 - start your conductor by running: `npm run start:conductor-1`
 - once your ui loads, you should see the web sdk sign up/sign in modal, sign up and log in
  - notice in the console that you have generated keys and now have a new dna-instance
 - **manual update** : You will need to update your conductor config with the new dna-instance:
    - shut down your conductor only
    - update your instance with the new agent ID
    - start back up your conductor
    - refresh your browser page
 - notice all `get` calls to be successful
 - update your profile name and notice successful change
 - sign out using the button in the header
 - once logged out, you should notice the web sdk login module appear again
